modules/install/pages/install-platforms.adoc

```diff
@@ -24,7 +24,7 @@ Nodes in a Couchbase cluster should all be running on the same OS, and every eff
 | CentOS
 | 8.x (Deprecated)
```

Comment:
This should be removed per DOC-7997

Reply:
If you're wondering why CentOS8 was removed before CentOS7, it's because the OS itself is not longer being maintained https://blog.centos.org/2020/12/future-is-centos-stream/